domains_plugin.php

```diff
@@ -396,6 +401,84 @@ private function upgrade1_6_0()
         }
     }
 
+    /**
+     * Update to v1.6.1
+     */
+    private function upgrade1_6_1()
```

Comment:
This upgrade causes a fatal error if there are already terms set

```
Uncaught Error, code 0 in ...\app\models\package_options.php on line 260 Message: Cannot use object of type stdClass as array
```

I think if there are any pricings set for a feature we should skip the edit for that feature